src/openrct2/ride/Ride.h

```diff
@@ -993,6 +993,7 @@ struct rct_ride_properties
 #define TURN_MASK_4_PLUS_ELEMENTS 0xF800
 
 #define CONSTRUCTION_LIFT_HILL_SELECTED 1
+#define CONSTRUCTION_INVERTED_TRACK_SELECTED 2
```

Comment:
Why not a `constexpr int32_t` instead?

Reply:
Should also use the `1 << 1` or `0b00000001` notation, I think.